README.md

```diff
@@ -13,7 +13,23 @@
 
 <!-- usage note: edit this section. -->
 
-A friendly guide to GitHub and all the things you can **currently** do with it. Very few papers focus on GitHub as a tool for collaboration. We will also mention where GitHub falls short as a tool for collaboration.
+A friendly guide to GitHub and all the things you can **currently** do with it. Very few papers focus on GitHub as a tool for collaboration. We will also mention where GitHub falls short as a tool for collaboration.  
+
+## Important links and dates
+In this section you'll find a few important links to help us keep track of documents we use outside of the GitHub ecosystem.  This includes a google slide deck where we are working on figures, the original outline we made for the manuscript in HackMD, as well as meeting notes.
+
+**Links**
```

Comment:
```suggestion
**Links**
[Figure brainstorming](https://docs.google.com/presentation/d/1SnAwK4XLlKf-XqGCqianW3-cGp3SlQYnCZ6ASJDKn_c/edit#slide=id.gf6fdd2a2d0_1_26)
```
I would add the original figure brainstorming google slides, so we have those at hand if needed!

Reply:
That's a good call, @LunaSare. Thanks for adding the link!